colors.d.ts

```diff
@@ -1,531 +1,531 @@
 type CalciteColors = {
```

Comment:
How is this type exactly used? Types don't emit code, so this wouldn't help getting the mapped values.

Reply:
@jcfranco this file was already there. I think the prettier just modified it. I'll update it to be generated by TSC though.